Gruntfile.js

```diff
@@ -15,4 +20,6 @@ module.exports = function (grunt) {
   require('load-grunt-tasks')(grunt);
   // load local Grunt tasks
   grunt.loadTasks('tasks');
+  // load third-party tasks
+  grunt.loadNpmTasks('grunt-mocha');
```

Comment:
With [the asked for change](https://github.com/mozilla/fxa-password-strength-checker/pull/41/files#r50235926), this line is no longer needed.